src/main/java/com/Elessar/app/client/HttpClient.java

```diff
@@ -0,0 +1,24 @@
+package com.Elessar.app.client;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpContent;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.http.protobuf.ProtoHttpContent;
+import com.google.protobuf.GeneratedMessageV3;
+
+import java.net.URL;
+
+/**
+ * Created by Hans on 1/19/19.
+ */
+public class HttpClient {
```

Comment:
Some high level notes:

- HttpTransport and HttpRequestFactory should each be a field of HttpClient instead of being new-ed for each method call. This is mentioned by the doc:
https://developers.google.com/api-client-library/java/google-http-java-client/reference/1.20.0/com/google/api/client/http/HttpTransport
https://developers.google.com/api-client-library/java/google-http-java-client/reference/1.20.0/com/google/api/client/http/HttpRequestFactory
Notice that the doc doesn't explicitly mention you should reuse instance of HttpRequestFactory, but we can infer that from "Thread-safe light-weight HTTP request factory layer ..."

- We want to have 2 methods for now and MyClient should always delegate requests to HttpClient.
public HttpResponse get(URL url);
public HttpResponse post(URL url, MessageLite);

- If you read the first note above carefully then you will already realize that MyClient should have a field of type HttpClient, which is passed in from MyClient's constructor.